### PR TITLE
Add join/unjoin documentation for media_player

### PR DESCRIPTION
**Pull request by klada**
## Proposed change

Document the new media_player services `join` and `unjoin`.

The first integration which will be using this is HEOS: https://github.com/home-assistant/core/pull/32568

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/41193
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/media_player.markdown [hunk 1]
```diff
@@ -156,6 +156,21 @@ Currently only supported on [Sonos](/integrations/sonos), [Spotify](/integration
 | `entity_id`            |      yes | Target a specific media player. For example `media_player.kitchen`|
 | `repeat`               |       no | `off`/`all`/`one` for setting repeat mode            |
 
+#### Service `media_player.join`
+
+Allows to group media players together for synchronous playback. Only works on supported multiroom audio systems.
+
+| Service data attribute | Optional | Description                                          |
+| ---------------------- | -------- | ---------------------------------------------------- |
+| `entity_id`            |       no | The media player entity whose playback will be expanded to the players specified in `group_members`.  |
```
Comment (frenck) [inline source/_integrations/media_player.markdown]: This service is implemented using `async_register_entity_service`, making `entity_id` optional.

```suggestion
| `entity_id`            |     yes | The media player entity whose playback will be expanded to the players specified in `group_members`.  |
```

File: source/_integrations/media_player.markdown [hunk 2]
```diff
@@ -156,6 +156,21 @@ Currently only supported on [Sonos](/integrations/sonos), [Spotify](/integration
...
+| `group_members`        |       no | The player entities which will be synced with the playback from `entity_id`.  |
+
+#### Service `media_player.unjoin`
+
+| Service data attribute | Optional | Description                                          |
+| ---------------------- | -------- | ---------------------------------------------------- |
+| `entity_id`            |       no | Unjoin this media player from any player groups.     |
```
Comment (frenck) [inline source/_integrations/media_player.markdown]: same as above. 

```suggestion
| `entity_id`            |     yes | Unjoin this media player from any player groups.     |
```

**Review by frenck — CHANGES_REQUESTED** [review]
2 small suggestions 👍

File: source/_integrations/media_player.markdown (hunk 1 above)
Comment (klada) [reply source/_integrations/media_player.markdown]: OK, thanks. I have updated the docs.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**